FxA/FxA/CHUtils.m

```diff
@@ -31,7 +31,7 @@ + (void)initialize {
 
 + (CHNumber *)generatePrimeOfLength:(int)numBits safe:(BOOL)safe add:(CHNumber *)add remainder:(CHNumber *)rem {
 	CHNumber * result = [CHNumber number];
-	BN_generate_prime([result bigNumber], numBits, safe, [add bigNumber], [rem bigNumber], NULL, NULL);
+	BN_generate_prime_ex([result bigNumber], numBits, safe, [add bigNumber], [rem bigNumber], NULL);
```

Comment:
Done. 

This is deadcode: we only use `[CHUtils primesUpTo:]`, which is only used by `[CHNumber factors]`.

Proposed: delete `factors` and CHUtils.m and CHUtils.h.

https://github.com/davedelong/CHMath